model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/client/entity/JpaClientMetadata.java

```diff
@@ -18,9 +18,18 @@
 
 import java.io.Serializable;
 import org.keycloak.models.map.client.MapClientEntityImpl;
+import org.keycloak.models.map.common.DeepCloner;
 
 public class JpaClientMetadata extends MapClientEntityImpl implements Serializable {
 
+    public JpaClientMetadata(DeepCloner cloner) {
+        super(cloner);
```

Comment:
This needs extra care. We do not want to serialize `cloner` into database. Is this handled in this way, @vramik?

Reply:
It should be handled by https://github.com/keycloak/keycloak/blob/main/model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/jsonb/JsonbType.java#L72

Reply:
The `cloner` is there anyway, the change is that it is possible to use something different than `DUMB_CLONER`. Not sure it makes sense though, I just needed to add a constructor with DeepCloner so that it is possible to use it in `DeepCloner.Builder#constructor`. If there is something wrong we can revert.

Reply:
The constructor is not used from anywhere at the moment. I'm not sure about future use cases, but right now the class is instantiated only with `DUMP_CLONER`.